clippy.toml

```diff
@@ -0,0 +1 @@
+msrv = "1.63.0"
```

Comment:
Newline at EOF?

Also, is this not inferred from the `Cargo.toml` files it is applied to? We have a `rust_version` field in those...

Reply:
Indeed. This can be removed then.